builtin/settingtypes.txt

```diff
@@ -1413,12 +1413,6 @@ curl_file_download_timeout (cURL file download timeout) int 300000
 #    Makes DirectX work with LuaJIT. Disable if it causes troubles.
 high_precision_fpu (High-precision FPU) bool true
 
-#    Changes the main menu UI:
```

Comment:
You can separate it for another PR. It is deprecated and not used due to criticism of all Android players. But this is another issue.

Reply:
I already have 2 PRs running (this and the waypoint one). This is part of mainmenu improvements and therefore included.